### PR TITLE
feat(commands): setup minimal yargs commands

### PR DESCRIPTION
**Pull request by BioPhoton**
This PR implements a minimal set of commands.

by default, it restores the old behavior

### CODE REVIEW

File: packages/deprecation-crawler/tests/index.test.ts [hunk 1]
```diff
@@ -76,7 +76,7 @@ test('sandbox', async () => {
       (d) => d.group === 'multiple-string-patterns-at-once'
     )
   ).toHaveLength(0);
-  // BUG: this has to have 0 hits
+  // @TODO BUG: this has to have 0 hits
```
Comment (BioPhoton) [inline packages/deprecation-crawler/tests/index.test.ts:79]: Should we do something here?

File: packages/deprecation-crawler/src/lib/utils.ts [hunk 2]
```diff
@@ -139,7 +139,7 @@ export function getVerboseFlag() {
  * Check for version params from cli command
  */
 export function getVersion() {
-  const argPath = getCliParam(['next-version', 'v']);
+  const argPath = getCliParam(['next-version', 'n']);
```
Comment (BioPhoton) [inline packages/deprecation-crawler/src/lib/utils.ts:142]: v conflicts with --verbose alias -v

File: packages/deprecation-crawler/tests/index.test.ts (hunk 1 above)
Comment (timdeschryver) [reply packages/deprecation-crawler/tests/index.test.ts:79]: Yes, probably 😅
There's something wrong with the regex I think.

File: packages/deprecation-crawler/src/lib/commands.ts [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  {
+    command: 'group',
+    description: 'Run the group processor',
+    module: {
+      handler: async (argv) => {
+        if (argv.verbose) console.info(`run grouping as a yargs command`);
+        const config = await getConfig();
+        const tasks = [loadExistingDeprecations, addGroups, generateOutput];
+
+        // Run all processors
```
Comment (timdeschryver) [inline packages/deprecation-crawler/src/lib/commands.ts:51]:
```suggestion
```

**Review by timdeschryver — APPROVED** [review]
This is nice!!

File: packages/deprecation-crawler/tests/index.test.ts (hunk 1 above)
Comment (BioPhoton) [reply packages/deprecation-crawler/tests/index.test.ts:79]: What should I check? Give me a request and I solve it.

File: packages/deprecation-crawler/src/lib/commands.ts (hunk 3 above)
Comment (BioPhoton) [reply packages/deprecation-crawler/src/lib/commands.ts]: closed as outdated

**Comment by BioPhoton** [comment]
After your positive feedback, I move a lot of other things to structure the code a bit more and group responsibilities.
As an excuse for the big PR I blame the situation this is the only PR open ATM. :D

**Comment by BioPhoton** [comment]
Ok, i messed it up :D sry. fix it soon! ;)

**Comment by BioPhoton** [comment]
Fixed :D

File: packages/deprecation-crawler/tests/index.test.ts (hunk 1 above)
Comment (timdeschryver) [reply packages/deprecation-crawler/tests/index.test.ts:79]: See #94

**Pull request merged**